hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/HddsUtils.java

```diff
@@ -215,6 +215,26 @@ public static OptionalInt getHostPort(String value) {
     }
   }
 
+  /**
+   * Retrieve a number, trying the supplied config keys in order.
+   * Each config value may be absent
+   *
+   * @param conf Conf
+   * @param keys a list of configuration key names.
+   *
+   * @return first number found from the given keys, or absent.
+   */
+  public static OptionalInt getNumberFromConfigKeys(
```

Comment:
Would be good to include a unit test for this.

Reply:
Thanks @kerneltime, I've updated the `TestHddsUtils` to include a unit test for `getNumberFromConfigKeys` in latest commit.